utility/cli_message_examples_reprex.md

```diff
@@ -77,6 +118,20 @@ read_redcap(redcap_uri, classic_token, suppress_redcapr_messages = c(TRUE, TRUE)
 #>   is not a valid value
 #> ! Must have length 1, but has length 2
 
+# data access groups
+
+read_redcap(redcap_uri, classic_token, export_data_access_groups = TRUE)
+#> Error in `read_redcap()`:
+#> ✖ Data access groups requested, but none found.
+#> ℹ REDCapTidieR did not detect expected columns for a project using DAGs.
+
+# surveys
+
+read_redcap(redcap_uri, longitudinal_token, export_survey_fields = TRUE)
+#> Error in `read_redcap()`:
+#> ✖ Project survey fields requested, but none found.
+#> ℹ REDCapTidieR did not detect expected columns for a survey-enabled project.
+
```

Comment:
(i) Are you sure the project has at least one instrument configured as a survey?